src/components/Customer.js

```diff
@@ -1,8 +1,13 @@
 import React, { Component } from "react";
+import { Link, Route } from "react-router-dom";
 import { Table } from "react-bootstrap";
 import { Button, ButtonToolbar } from "react-bootstrap";
 import { AddCustModal } from "./AddCustModal";
 import { EditCustModal } from "./EditCustModal";
+import { BrowserRouter as Router } from "react-router-dom";
+//import { Redirect } from "react-router-dom";
+//import { Redirect } from "react-router";
+import { FbAuth } from "./FbAuth";
 export class Customer extends Component {
   constructor(props) {
     super(props);
```

Comment:
do not use 'cust' as variable-name (in state). Use full word 'customers' for better readability